cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```
@@ -351,6 +353,28 @@ spec:
           securityContext:
             runAsUser: 0
 {{ end }}
+        {{- if .Values.kubecostModel.caCertsSecret }}
+        - name: "sslsetup"
```

```suggestion
        - name: "update-ca-trust"
```
what do you think of changing the container name?

---

agree